Android/app/build.gradle

```
@@ -3,13 +3,13 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 33
+    compileSdkVersion 34
 
     defaultConfig {
         applicationId "live.ditto.inventory"
         minSdkVersion 26
-        targetSdkVersion 33
-        versionCode 55
+        targetSdkVersion 34
+        versionCode 56
         versionName "1"
```

```suggestion
        versionName "1.0.1"
```

Let's update the version name as well